VotingApplication/VotingApplication.Web.Tests/E2E/BasicPollTests.cs

```diff
@@ -179,22 +87,26 @@ public void AfterVoting_VoteIsRemembered()
                     using (var context = new TestVotingContext())
                     {
                         CreatePoll(context);
-                        GoToUrl(driver, PollUrl);
+                        GoToUrl(driver, PollVoteUrl);
 
-                        IWebElement voteButtons = FindElementById(driver, "vote-button");
-                        voteButtons.Click();
+                        IReadOnlyCollection<IWebElement> choices = driver.FindElements(NgBy.Binding("choice.Name"));
+                        choices.First().Click();
+
+                        IWebElement voteButton = FindElementById(driver, "vote-button");
+                        voteButton.Click();
 
                         NavigateBackToVotePage(driver);
 
-                        IWebElement selectedChoice = driver.FindElement(By.CssSelector(".selected-choice"));
+                        IWebElement selectedChoice = driver.FindElement(By.CssSelector(".md-ripple-visible"));
```

Comment:
I know this was done a bit hackily before, but I am wondering if there is a way of doing this that doesn't involve css class selectors.